lib/chai/utils/proxify.js

```diff
@@ -32,6 +39,12 @@ module.exports = function proxify (obj) {
       if (typeof property === 'string' &&
           config.proxyExcludedKeys.indexOf(property) === -1 &&
           !Reflect.has(target, property)) {
+        // Special message for invalid property access of non-chainable methods.
+        if (nonChainableMethodName) {
+          throw Error('Invalid use of Chai method: ' + nonChainableMethodName +
+            '. Invocation required.');
+        }
```

Comment:
Okay, I like the idea of "See docs...". I agree that we dont want to present wrong info, but I also what to give users a chance to fix without immediately finding a support channel (like our issue tracker 😵)